eth/stagedsync/stage_tevm.go

```diff
@@ -142,6 +145,13 @@ func transpileBatch(logPrefix string, s *StageState, fromBlock uint64, toBlock u
 		}
 	}
 
+	// Prune TEVM statuses if needed
+	if cfg.pruningDistance > 0 {
```

Comment:
This needs to be unconditional, it is always required, see how it is done in `stage_call_traces`

Reply:
Ok. Will do.